src/client.js

```diff
@@ -93,7 +93,35 @@ export function bootstrap() {
     )
     : new URL(link.getAttribute("href"));
   const client = new Client(initialURL.href);
-  // Register a global listener for history updates.
+
+  // Add click event listener to div#app
+  const appDiv = document.getElementById("app");
+  console.assert(!!appDiv, "bootstrap error: missing div#app element");
+  appDiv.addEventListener("click", (event) => {
+    // Check if the target is an anchor element
+    if (!(event.target instanceof HTMLAnchorElement)) return;
+
+    const anchor = event.target;
+    const href = anchor.href; // Safe to access href directly
+
+    // Check if href is a relative URL
+    const isRelative = href &&
+      (href.startsWith("./") || href.startsWith("/") ||
+        !/^(?:[a-z]+:)?\/\//i.test(href));
+    if (!isRelative) return;
+
+    // Check if anchor has a type attribute (opt-out)
+    if (anchor.hasAttribute("type")) return;
+
+    // Prevent default behavior and stop propagation
+    event.preventDefault();
+    event.stopPropagation();
+
+    // Call the client's follow function
+    client.follow(href, {});
```

Comment:
nit: no need to pass the default value IMO

```suggestion
    client.follow(href);
```